specs/protocol/configurability.md

```diff
@@ -0,0 +1,71 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Protocol Parameters](#protocol-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are three categories of OP Stack configuration options:
+
+- **Protocol Parameters**: Parameters and properties of the chain that may
```

Comment:
We should differentiate between consensus and policy here. Consensus is enforced while policy includes the set of possible behaviors within consensus. An example of policy is the tx p2p gossip size limit, where consensus will accept a tx of any size given it fits in the block gas limit